src/screens/WaitTimeScreen.tsx

```diff
@@ -1,42 +1,471 @@
+              <StyledText style={styles.stepDescriptionText}>
+                {item.description}
+              </StyledText>
+            </View>
+          ))}
+        </View>
+        <View style={styles.stepScrollDivider} />
+        <ScrollView
+          refreshControl={
+            <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
+          }
+        >
+          <List>
+            {poiData.map((poi: any, index: any) => (
+              <Item
+                key={index}
+                onPress={() => {
+                  navigation.navigate(LOCATION_DETAILS, {
+                    location: poi.location,
+                  });
+                }}
+                extra={
+                  <View>
+                    <Brief style={styles.poiWaitTimeText}>
+                      {poi.waitTime + " mins"}
+                    </Brief>
+                  </View>
+                }
+                arrow="horizontal"
+                thumb={
+                  <View style={styles.poiIcon}>
+                    {renderIcon(poi.type, theme.name)}
+                  </View>
+                }
+              >
+                <StyledText style={styles.poiDistanceText}>
+                  {poi.distance + " km"}
+                </StyledText>
+                <StyledText>{poi.location}</StyledText>
+                <StyledText style={styles.poiLastUpdatedText}>
+                  {renderLastUpdated(poi.lastUpdated)}
+                </StyledText>
+              </Item>
+            ))}
+          </List>
+        </ScrollView>
+      </View>
+    </TouchableWithoutFeedback>
   );
 };
 
 const styles = StyleSheet.create({
-  huge: {
-    fontSize: 20,
+  searchContainer: {
+    paddingTop: 20,
+    backgroundColor: "#FFFFFF",
   },
-  container: {
-    flex: 1,
-    backgroundColor: "#fff",
+  searchBar: {
+    borderRadius: 6,
+    backgroundColor: "#F5F5F5",
+  },
+  search: {
+    backgroundColor: "#F5F5F5",
+  },
+  sortContainer: {
+    paddingTop: 20,
+  },
+  tags: {
+    paddingRight: 6,
+  },
+  sortByTag: {
+    borderRadius: 24,
+    width: "20%",
+    height: 34,
+    backgroundColor: "#FFFFFF",
+    marginRight: 6,
+  },
+  sortByText: {
+    fontSize: 15,
+    textAlign: "center",
+  },
+  modal: {
+    height: "100%",
+    width: "100%",
+    backgroundColor: "rgba(0,0,0,0)",
+  },
+  modalBackground: {
+    height: "100%",
+    width: "100%",
+    justifyContent: "flex-end",
+  },
+  modalButtons: {
+    color: "#1677FF",
+  },
+  modalHeader: {
+    width: "100%",
+    backgroundColor: "#ffffff",
+    borderTopLeftRadius: 10,
+    borderTopRightRadius: 10,
+    paddingBottom: 10,
+    paddingHorizontal: 20,
+    paddingTop: 20,
+  },
+  modalActionRow: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    paddingBottom: 20,
+  },
+  sortButtons: {
+    borderColor: "#FFFFFF",
+    alignItems: "flex-start",
+  },
+  checkIconTime: {
+    paddingLeft: "85%",
```

Comment:
The padding can be a fixed number. It's often easier to understand and work with than a percentage value. Stuff like 76% padding can start to get confusing and is often unnecessary for static containers.

Reply:
I tried fix number, for some reason on larger screens it wasn't getting proportioned, so had to use percentages

Reply:
Okay willing to leave this is it doesn't cause problems.